app/app.js

```diff
@@ -90,24 +98,26 @@ async function init() {
         block.appendChild(popChip);
       }
 
-      item.appendChild(block);
-      flowNode.appendChild(item);
+      panel.appendChild(block);
+      flowNode.appendChild(panel);
     } else {
       detailPanelNode.heading = handleCasing(attributes["NAME"]);
       document.getElementById(
         "detail-chip-type"
       ).innerText = `type: ${handleCasing(attributes["NAICS_DESC"])}`;
+      document.getElementById("detail-item-website").innerText = `Learn more`;
       document.getElementById(
-        "detail-chip-website"
-      ).innerText = `website: ${attributes["WEBSITE"]}`;
+        "detail-item-website"
+      ).href = `http://${attributes["WEBSITE"]}`;
+
       document.getElementById(
         "detail-chip-pop"
       ).innerText = `population: ${attributes["POPULATION"]}`;
     }
     view.goTo(
       {
         center: [result.geometry.longitude, result.geometry.latitude],
-        zoom: 13,
+        zoom: 10,
```

Comment:
Julie wanted this at 13 last time we spoke. Is this ok @juliepowell ?

Reply:
Happy to revert.